tests/cassettes/test_send_ping.yaml

```diff
@@ -0,0 +1,27 @@
+interactions:
```

Comment:
This is one of the possible states if you have an invalid certificate. If you however do present a proper certificate and auth, you will receive a 200 (or 201 don't remember).

In short, this is a valid mocked result, but only for a "failed" ping so to speak.

Reply:
Updated with proper certificate! Both cases are covered now.